common/extensions/api/_ipfs_api_features.json

```diff
@@ -54,5 +54,21 @@
     "contexts": ["blessed_extension"],
     "dependencies": [],
     "allowlist": ["780BF954C0F7C586EA9662D4F967771F49CC2114", "FF32507DC3DB5DFFD1D6733187C84D4B74713D63", "A321D47A2B4CA86898167A55CA8B2E02385EA7CD"]
+  }],
+  "ipfs.getSettings": [{
+    "channel": "stable",
+    "contexts": ["blessed_extension"],
+    "dependencies": ["permission:ipfs"]
+  },{
+    "channel": "stable",
+    "contexts": ["blessed_extension"],
+    "dependencies": [],
+    "allowlist": ["780BF954C0F7C586EA9662D4F967771F49CC2114", "FF32507DC3DB5DFFD1D6733187C84D4B74713D63", "A321D47A2B4CA86898167A55CA8B2E02385EA7CD"]
```

Comment:
do we still need to be allowlisting the extension ids for ipfs-companion-beta and brave-shields?

Reply:
Seems brave-shields could be removed